tests/dialects/test_tsql.py

```diff
@@ -753,6 +753,15 @@ def test_types_date(self):
             },
         )
 
+        # percision of TIMESTAMP_NTZ 0 to 9 (same as TIMESTAMP)
+        # percision of DATETIME2 0 to 7
+        self.validate_all(
+            "CREATE TABLE t (col1 TIMESTAMP_NTZ(2))",
+            write={
+                "tsql": "CREATE TABLE t (col1 DATETIME2(2))",
+            },
+        )
+
```

Comment:
The path of interest here is Snowflake -> T-SQL, so we should make that clear:

```suggestion
        self.validate_all(
            "CREATE TABLE t (col1 DATETIME2(2))",
            read={
                "snowflake": "CREATE TABLE t (col1 TIMESTAMP_NTZ(2))",
            },
            write={
                "tsql": "CREATE TABLE t (col1 DATETIME2(2))",
            },
        )
```

Reply:
Good catch, thanks.